### PR TITLE
Handle new neovim UI events.

### PR DESCRIPTION
**Pull request by rygwdn**
Replace cursor_{on,off} by busy_{stop,start}. Change done in: neovim/neovim@c546875daf36936b9a6c0886a71c9edd1fdae6db.

### CODE REVIEW

File: src/nvim/ui.coffee [hunk 1]
```diff
@@ -206,8 +206,13 @@ class UI extends EventEmitter
   nv_cursor_goto: (row, col) ->
     @cursor_row = row
     @cursor_col = col
-  nv_cursor_off: -> @cursor.style.display = 'none'
-  nv_cursor_on: -> @cursor.style.display = 'block'
+
```
Comment (coolwanglu) [inline src/nvim/ui.coffee:222]: The handlers are ordered in alphabetical order. And the `nv_cursor_*` functions are grouped together without blank lines in between.

**Comment by coolwanglu** [comment]
Thanks. Please fix the order of the handlers.

**Comment by coolwanglu** [comment]
Also as neovim is still in alpha, we may just follow the latest version :)

**Comment by rygwdn** [comment]
@coolwanglu I removed the old handlers, and re-ordered the new ones.

File: src/nvim.css [hunk 2]
```diff
@@ -25,11 +25,18 @@ canvas {
   position: absolute;
   box-sizing: border-box;
   border: 1px solid;
+
+  -webkit-animation: 1s blink step-end infinite;
```
Comment (coolwanglu) [inline src/nvim.css]: Why adding this line?

**Comment by coolwanglu** [comment]
Seems that you included the commits for blinking cursors, can you remove that and probably do a rebase/squash?

**Comment by rygwdn** [comment]
@coolwanglu fixed and squashed

**Comment by coolwanglu** [comment]
Thanks!

**Pull request merged**